web/src/components/data-entities/site/SiteList.js

```diff
@@ -142,9 +142,7 @@ const SiteList = () => {
             filter={false}
             resizable={false}
             sortable={false}
-            valueFormatter={(e) => {
-              return e.data.isActive ? '' : 'Delete';
-            }}
+            cellRenderer={() => <Delete/>}
```

Comment:
This should probably be ` cellRenderer={(e) => e.data.isActive ? <></> : <Delete/>}`

Reply:
updated